---
permalink: /
title: "Sergio's Personal Page"
excerpt: "Computational Biologist | Scientist at dsm-firmenich | PhD Systems Medicine"
author_profile: true
redirect_from: 
  - /about/
  - /about.html
---

# About

I am a computational biologist currently working in the Data Science team at dsm-firmenich. My work focuses on integrating multi-omics datasets and phenotypic readouts to support and develop products in the areas of healthy aging, cosmetics, and beauty.  

Previously, I obtained a PhD at the University Medical Center Groningen, where I explored the drivers of interindividual variation in immunological and metabolic biomarkers in large, well-phenotyped population cohorts. I then completed a short postdoc at [VIB](https://cfm.sites.vib.be/en) in Leuven, Belgium, focusing on understanding temporal variation in subspecies diversity of the gut microbiome in both population-based and intervention settings.

---

# CV

## May 2025 – Present: Associate Scientist, Data Science, dsm-firmenich
I joined dsm-firmenich in a scientific role with the goal of translating microbiome insights into products with real-world impact.

---

## July 2024 – May 2025: Postdoctoral Researcher, VIB/KU Leuven
I worked as a postdoctoral researcher in the [Raes Lab](https://raeslab.sites.vib.be/en) at VIB.  
My focus was on understanding subspecies-level variation in the gut microbiome in both population and intervention studies. This included analyses of large cohort-based longitudinal studies and *ex-vivo* fermentation datasets.

---

## May 2023 – September 2024: Visiting Scholar, University of Trento
I joined the [Computational Metagenomics Group](http://segatalab.cibio.unitn.it/) led by [Nicola Segata](https://scholar.google.com/citations?user=ZXjO-Q4AAAAJ) in Trento, Italy.  
My research focused on integrating large-scale metagenomic data at subspecies resolution to study associations with human health and disease.  

This work was supported by EMBO, published in [Cell](https://www.cell.com/cell/fulltext/S0092-8674(25)00416-7), and featured in the media ([MicrobiomePost](https://microbiomepost.com/tiny-differences-in-gut-bacteria-may-influence-health-and-disease/), [NutraIngredients](https://www.nutraingredients.com/Article/2025/05/13/gut-microbiome-study-uncovers-strain-level-geographic-differences-and-disease-links/)).

---

## January 2020 – June 2024: PhD Studies, University Medical Center Groningen
PhD in the Microbiome Group at the [University Medical Center Groningen](https://www.umcg.nl) under Prof. [Jingyuan Fu](https://www.rug.nl/research/genetics/staff/jingyuan-fu) [[Google Scholar](https://scholar.google.nl/citations?user=7OHBkYMAAAAJ&hl=en)] and Prof. [Sasha Zhernakova](https://www.rug.nl/research/genetics/staff/alexandra-(sasha)-zhernakova) [[Google Scholar](https://scholar.google.com/citations?user=lrt2jA8AAAAJ&hl=en)].  

I worked closely with the Lifelines project, a three-generation population cohort from Northern Netherlands. By analyzing gut microbiome, genetics, lifestyle, diet, immune, and metabolic data, I explored diverse questions around human health and disease.  

Our work included:  
- Antibody repertoires in the [general population](https://www.cell.com/immunity/fulltext/S1074-7613(23)00171-1) and in [inflammatory bowel disease](https://www.cell.com/immunity/fulltext/S1074-7613(23)00185-1?uuid=uuid%3A656c1b70-dec5-4cc2-9cd1-7f70228b4392)  
- Biological aging, such as [telomere length variation](https://www.nature.com/articles/s42003-022-03521-7)  
- Subspecies-level microbiome variation, including [structural variation](https://www.nature.com/articles/s41586-023-06893-w) and [phylogenetic associations](https://www.cell.com/cell/fulltext/S0092-8674(25)00416-7)

---

## September 2018 – 2020: MSc Studies, University of Copenhagen
MSc in Bioinformatics at the [University of Copenhagen](https://www.ku.dk/english/), Denmark.  
My thesis research was part of the Bird 10,000 Genomes Project at the [Centre for Biodiversity Genomics](http://zhanggjlab.org) under Prof. Guojie Zhang [[Google Scholar](https://scholar.google.com/citations?user=hcIDquMAAAAJ&hl=en)].  
I was part of the main manuscript for the [second phase of the project](https://www.nature.com/articles/s41586-020-2873-9), and as part of my thesis, we published a study looking into [frame-shift variation in mitochondrial genes](https://academic.oup.com/gigascience/article/10/1/giaa161/6104268)

During my MSc, I also worked part-time as a bioinformatician at [Evaxion Biotech](https://www.evaxion-biotech.com) and at Rigshospitalet’s [Genomic Medicine Department](https://www.rigshospitalet.dk/afdelinger-og-klinikker/diagnostisk/genomisk-medicin/Sider/default.aspx). During my work at Genomic Medicine, we set up the infrastrucutre for routine NGS-based diagnosis of several diseases. We created guidelines and [benchmarks for variant calling](https://www.mdpi.com/2072-6694/13/24/6283) which were used daily at the hospital.

---

## September 2013 – 2018: BSc Studies, Technical University of Valencia
BSc in Biotechnology (Human and Biomedical track) from the [Technical University of Valencia](http://www.upv.es).  

During my studies, I completed two international exchanges:  
- [Jagiellonian University](https://en.uj.edu.pl/en_GB/start), Kraków, Poland (9 months)  
- [Tsinghua University](https://www.tsinghua.edu.cn/en/), Beijing, China (6 months)  

After completing my degree, and before starting my MSc, I worked as an intern in [JL Micol’s group](https://genetics.edu.umh.es).
